src/main/java/seedu/address/storage/XmlAdaptedActivity.java

```diff
@@ -105,7 +106,8 @@ public Activity toModelType() throws IllegalValueException {
         final Remark remark = new Remark(this.remark);
 
         final Set<Tag> tags = new HashSet<>(personTags);
-        return new Activity(name, dateTime, remark, tags);
+        //Changed Here.
```

Comment:
@karenfrilya97 please add your code here

Reply:
I think I wanna make this method abstract here